apps/web/server/lib/ssr.ts

```diff
@@ -28,6 +28,8 @@ export async function ssrInit(context: GetServerSidePropsContext) {
   await ssr.viewer.public.i18n.fetch();
   // So feature flags are available on first render
   await ssr.viewer.features.map.prefetch();
+  // Provides a better UX to the users who have already upgraded.
+  await ssr.viewer.teams.hasTeamPlan.prefetch();
```

Comment:
There was unnecessary skeleton loader when I have already upgraded. 
I think this change would improve experience for all features that are behind UpgradeTip